### ISSUE TITLE
Does not work with Windows 10 Home

### ISSUE THREAD

**Issue by JonnyMako** [onlineth/Spotlight-Desktop#17]
Does this work with Windows 10 Home? I can;t get it to work. Is there any way for me to get some error logs? thanks.

(works fine on my other machine which is Windows 10 Pro)

**Comment by onlineth** [comment]
What is not working? An incorrect image displaying? No image being set? Black background? If you could follow this guide and let me know what turns up - https://github.com/onlineth/Spotlight-Desktop/wiki/Backtrace-Spotlight-Image

**Comment by JonnyMako** [comment]
Thanks for the reply. The wallpaper was not changing. I followed the instructions on the link you provided, and could not find the lock screen image in the AppData...Assets location. After doing a bit of research, it turned out my spotlight lock screen was stuck. I was able to fix that problem as outlined in this link:

https://answers.microsoft.com/en-us/windows/forum/windows_10-start-winpc/windows-spotlight-lock-screen-is-stuck-on-the-same/354224ea-203c-4b27-9ec9-0343bd100258

Now Spotlight-Desktop is picking up the image from lock screen and displaying it on the desktop!
